tests/mitoai_ui_tests/aiInlineCompleter.spec.ts

```diff
@@ -4,7 +4,7 @@ import { PromiseDelegate } from "@lumino/coreutils";
 const GHOST_SELECTOR = ".jp-GhostText";
 
 test.describe("first time setup", () => {
-  test("should ask the user to activate the inline completion", async ({
+  test.skip("should ask the user to activate the inline completion", async ({
```

Comment:
Sounds good. We'll fix this up in a separate PR soon.

Reply:
Thanks! Created an issue so we don't loose track: https://github.com/mito-ds/mito/issues/1426